### PR TITLE
Feature/meta as a service

### PR DESCRIPTION
**Pull request by dcz**

### CODE REVIEW

File: config/services.yml [hunk 1]
```diff
@@ -6,16 +6,20 @@
...
+            - %core.php_ext% 
+    phpbbseo.meta.listener:
+        class: phpbbseo\meta\event\listener
+        arguments:
+            - @phpbbseo.meta.core
+            - @dbal.conn
         tags:
-            - { name: event.listener }
-
+            - { name: event.listener }
```
Comment (carlo94it) [inline config/services.yml]: A new empty line is necessary at the end of file.

File: config/services.yml [hunk 2]
```diff
@@ -6,16 +6,20 @@
...
         arguments:
             - @config
             - @template
             - @user
-            - @dbal.conn
+            - @symfony_request
             - %core.root_path%
-            - %core.php_ext%
+            - %core.php_ext% 
+    phpbbseo.meta.listener:
```
Comment (carlo94it) [inline config/services.yml:20]: Add a new empty line between services.

File: config/services.yml [hunk 3]
```diff
@@ -6,16 +6,20 @@
...
-        class: phpbbseo\meta\event\listener
+services:
+    phpbbseo.meta.core:
+        class: phpbbseo\meta\core
         arguments:
             - @config
             - @template
             - @user
-            - @dbal.conn
+            - @symfony_request
```
Comment (carlo94it) [inline config/services.yml:16]: Do not use symfony_request service, but the phpBB one.

**Comment by carlo94it** [comment]
You need also to replace $config with $this->config in the constructor.

**Comment by dcz** [comment]
I only use the config variable in the constructor so it's not currently useful to keep it.

This is done like this to allow the mod to work without the usu acp.

File: config/services.yml (hunk 3 above)
Comment (dcz) [inline config/services.yml:16]: I inspired from path_helper. And it is actually a phpBB wrapper over symphony, so I guess it's ok.

**Pull request merged**